onebox.sh

```diff
@@ -182,6 +182,15 @@ get_branch()
     fi
```

Comment:
FYI: changes to onebox.sh will be reverted after https://github.com/Microsoft/edx-configuration/pull/59 is merged . the changes to bootstrap.sh will remain